readme.md

```diff
@@ -78,11 +106,22 @@ var Panda = function(name, age) {
   this.age = age;
   this.num_bamboo_eaten = 0;
 }
+
+Panda.prototype = {
+  eat_bamboo: function(){
+    self = this;
+    var eaten = self.num_bamboo_eaten;
+    self.num_bamboo_eaten = eaten + 1;
+  }
+}
+
 ```
```

Comment:
When writing out prototype methods we usually do it like the following so that our entire constructor is not overwritten.

```js
Panda.prototype.eat_bamboo = function() {
  this.num_bamboo_eaten = this.num_bamboo_eaten + 1;
}
```